src/contracts/GPv2Settlement.sol

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: LGPL-3.0-or-newer
-pragma solidity ^0.6.12;
+pragma solidity ^0.7.0;
```

Comment:
Shouldn't all these be ^0.7.5 as at least some of them won't be compilable with a version that is greater than 0.7.0 but less than 0.7.5?

Reply:
Very true! They should all be ^0.7.5